### DISCUSSION TITLE
Kubernetes/ Helm - will the community will like this contribution

### DISCUSSION THREAD

**Discussion by yuliyantsvetkov** [retypeapp/retype#400] [Ideas]
Howdy,

I am about to start creating Helm3 chart for Retype that will be quite the amazing thing with dynamic configuration, reloaders, and init containers for loading the configuration, and it will be cloud agnostic. It can provide the Retype Kubernetes community with a nice way to deploy Retype on the same.

Will you think open sourcing the Helm3 chart to the Retype community and adding it to the Installation steps, if liked, will be beneficial?

**Comment by geoffreymcgill** [comment]
Hi @yuliyantsvetkov. The chart would be awesome. I would love to try it.

As far as open sourcing the chart, this might be something better hosted by the community. We would have a difficult time maintaining the project. If you wanted to sign over the project and have it hosted within the https://github.com/retypeapp Organization, we might be able to do that too, although it would require a community maintainer.

Writing a guide would also be awesome and we would add it to the **Hosting** section of the documentation. Please feel free to submit a PR with the guide and we will review.

**Reply by yuliyantsvetkov** [reply]
Sorry for being so laggy on the response here @geoffreymcgill.

I would love to be the maintainer of the chart in the organization. First, let me create it, battle test it, and share it on public repo, so we can get the ball rolling.
